File: code-examples/java/currencies/update.java
HttpResponse<String> response = Unirest.put("http://api.marketcloud.it/v0/currencies/:id")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .body("{\"name\" : \"American Dollar\",\"formatting\" : \"$\" }")
  .asString();

File: code-examples/csharp/currencies/update.cs
var client = new RestClient("http://api.marketcloud.it/v0/currencies/:id");
var request = new RestRequest(Method.PUT);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"name\" : \"American Dollar\",\"formatting\" : \"$\" }", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);